lib/oli/delivery/singleton_depot_coordinator.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Oli.Delivery.SingletonDepotCoordinator do
```

Comment:
If we are using this module exclusively for testing purposes, should we move the file to the /test/support folder? And/or maybe extend the module doc to explain this.
```suggestion
defmodule Oli.Delivery.SingletonDepotCoordinator do
```